.github/workflows/python-build.yml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2022 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Check Python build
+
+permissions:
+  contents: read
+
+on:
+  push:
+    paths:
+      - '**.proto'
+      - 'gen/pb-python/**'
+  pull_request:
+    paths:
+      - '**.proto'
+      - 'gen/pb-python/**'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+      fail-fast: false
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@13ae5bb136fac2878aff31522b9efb785519f984 # v4.3.0
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Build
+        run: |
+          cd gen/pb-python
+          python -m venv env && source env/bin/activate
+          python -m pip install --upgrade pip
+          python -m pip install .[dev]
+
+          # NOTE: We run the sdist and wheel builds manually, since
+          # our package metadata includes relative paths (README.md and LICENSE)
+          # that are outside of the package root. As a result, our sdist
+          # cannot be used to produce a wheel in isolation.
+          python -m build --sdist --wheel
```

Comment:
N.B.: This isn't ideal, but it's the best we can do without adding our own copies of the top-level `README.md` and `LICENSE` to the Python package's directory. But perhaps that's what we want anyways, so that we can specialize the text of the `README.md` for the Python package, specifically?

Reply:
How about symlinks? 

+1 to "eventually we'll want our own README" but the license thing is unfortunate

Reply:
Ugh, forgot that `git` and hard links don't play along...

Reply:
If my understanding of `git` is right, having two copies of `LICENSE` in the repo isn't the worst thing -- the underlying blob storage should dedupe them. Still not the cleanest, but given that are you okay with the extra file @znewman01?

Reply:
Oh, missed your last comment. I'll go ahead then 🙂